modules/era/lua_dynamis/globals/era_dynamis_spawning.lua

```diff
@@ -1816,7 +1816,7 @@ xi.dynamis.setMobStats = function(mob)
         elseif mob:getFamily() == 358 then -- If Kindred
             mob:setMobLevel(math.random(77,80))
         else
-            mob:setMobLevel(math.random(75,77))
+            mob:setMobLevel(math.random(77,78))
```

Comment:
I think the level changes in this file are era+

Reply:
Removed.